src/main/src/main/java/org/geoserver/util/HTTPWarningAppender.java

```diff
@@ -0,0 +1,73 @@
+/* (c) 2018 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
```

Comment:
As a new file I think it should be 2021

Reply:
Not a new file, it's been renamed from NearestMatchWarningAppender, since it reports other types of warnings (default value) too now.